src/components/link/common/link.scss

```diff
@@ -20,28 +19,22 @@
   --sbb-link-gap: var(--sbb-spacing-fixed-1x);
 }
 
-:host([variant='block']) {
-  display: block;
-}
-
-// Using :where to be not more specific than inline variant
 :host([negative]) {
   @include sbb.link-variables--negative;
 }
 
-:host([variant='inline']) {
-  @include sbb.link-variables--inline;
-}
-
-:host([variant='inline'][negative]) {
-  @include sbb.link-variables--inline-negative;
-}
-
 :host([role='button']) {
   @include sbb.link-variables--button;
 }
 
-:is(.sbb-link, .sbb-link-button, .sbb-link-static) {
+:is(
```

Comment:
Yep, was thinking something similar. Since we don't have cross-component CSS rules I would go with `sbb-link-base` (in `link-base-element.ts`)

@DavideMininni-Fincons I guess you have the same thing for buttons

Reply:
I was wrong :/ we should use `sbb-action-base`

Reply:
I was looking at it too, for now I went for:

```
[class$="-link"],
[class$="-static"],
[class$="-button"] {
    ...
}
```